cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,80 @@
+describe('EngageSphere API - GET /customers', () => {
+  const baseUrl = 'http://localhost:3001/customers';
```

Comment:
Hardcoding the API URL would make running the same tests against a different environment hard.

I suggest that instead of doing that, we define the API URL as an `env` in the `cypress.config.js` file, which can be overwritten via the command line. This would allow us to run the same tests against different environments.